NitroxClient/GameLogic/Spawning/Metadata/StarshipDoorMetadataProcessor.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NitroxClient.Communication;
+using NitroxClient.Communication.Abstract;
+using NitroxModel.Core;
+using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.Spawning.Metadata
+{
+    public class StarshipDoorMetadataProcessor : GenericEntityMetadataProcessor<StarshipDoorMetadata>
+    {
+        public override void ProcessMetadata(GameObject gameObject, StarshipDoorMetadata metadata)
+        {
+            Log.Debug($"Received starshipDoor metadata change for {gameObject.name} with data of {metadata}");
```

Comment:
Should we keep this log ?

Reply:
Happy not to do this but as discussed before it was kept in to follow the SealedDoorMetadataProcessor Sunrunner wrote. Happy to remove as I agree possibly is just cluttering the logs

Reply:
Then maybe remove the log line from sun as well.